### DISCUSSION TITLE
Installation Fails on Pi W

### DISCUSSION THREAD

**Discussion by Dtwo-sys** [mKeRix/room-assistant#387] [Q&A]
I am trying to install room-assistant, following the instructions to the letter (I think) but after

wget -O - https://raw.githubusercontent.com/.../install-node-v.lts.sh | bash

I get

unlink: cannot unlink '/sbin/node': No such file or directory
unlink: cannot unlink '/sbin/npm': No such file or directory
ln: failed to create symbolic link '/sbin/node': File exists
ln: failed to create symbolic link '/sbin/npm': File exists

And if I carry on the end, nothing is found in the room-assistant folder other, other than the config file.
Any ideas?

thanks

**Comment by mKeRix** [comment]
If I remember correctly these warnings when running the NodeJS install script for Pi Zeros are nothing to worry about. If you can run `node -v` and get something with v10 or v12 back you're good.

The room-assistant folder is not supposed to contain anything other than the config. The application itself will be installed into a folder that is managed by `npm` (the installer tool). Did you extend your $PATH variable as well like written in the qucikstart guide? That is what makes room-assistant (should be in /opt/nodejs/bin if following the guide) available as a global command.

**Reply by Dtwo-sys** [reply]
Thank you taking time to reply, appreciated. Not sure what I missed but restarted afresh, continued past the warnings and it is running now.
